test/models/user_test.rb

```diff
@@ -1,13 +1,16 @@
 require 'test_helper'
 
 class UserTest < ActiveSupport::TestCase
+  attr_reader :user, :order
 
   def setup
-    @user = User.new(username: 'user',
+    @user = User.create(username: 'user',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.